pkg/docker/overview.js

```diff
@@ -234,86 +234,56 @@ define([
 
         // Render storage, throttle update on events
 
-        function render_storage() {
-            client.info().done(function(data) {
-                var resp = data && JSON.parse(data);
-                if (resp['Driver'] !== "devicemapper") {
-                    // TODO: None of the other graphdrivers currently
-                    // report size information.
-                    $('#containers-storage .bar').html();
-                    $('#containers-storage .data').html("Unknown");
-                }
+        $(client).on("info", function render_storage() {
+            if (client.info['Driver'] !== "devicemapper") {
+                // TODO: None of the other graphdrivers currently
+                // report size information.
+                $('#containers-storage .bar').html();
+                $('#containers-storage .data').html("Unknown");
+            }
 
-                var used;
-                var total;
-                var avail;
-                $.each(resp['DriverStatus'], function (index, value) {
-                    if (value && value[0] == "Data Space Total")
-                        total = value[1];
-                    else if (value && value[0] == "Data Space Used")
-                        used = value[1];
-                    else if (value && value[0] == "Data Space Available")
-                        avail = value[1];
-                });
+            var used;
+            var total;
+            var avail;
```

Comment:
I know this is legacy code, but when I read `avail`, the first two things that pop into my mind are the noun and verb `avail`, not the abbreviation of `available`.